foreman/data_refinery_foreman/surveyor/test_end_to_end.py

```diff
@@ -170,7 +170,7 @@ def test_array_express_redownloading(self):
             self.assertTrue(downloader_job.success)
 
             # Now we're going to delete one of the extracted files but not the other.
-            deleted_file = OriginalFile.objects.filter(is_archive=False)[0]
+            deleted_file = OriginalFile.objects.filter(is_archive=False).first()
```

Comment:
Hmm, looks like you've traded one potential exception for another.... `first()` can return None which doesn't have `delete_local_file()`, unless you made that change for other reasons.

Reply:
added an additional assertion below this line `self.assertIsNotNone(deleted_file)`